governance/addin-api/README.md

```diff
@@ -1,3 +1,9 @@
 # Governance Addin Api
 
-Governance Addin Api interface
+The addin API is for customising the governance structure of your DAO with plugins. 
+For example: 
+* Use [NFTs](https://docs.realms.today/DAO-Management/createing-DAOs/NFT-Community-DAO) as governance power.
```

Comment:
Please add links to NFT voter and VSR github repos

NFT Voter
https://github.com/solana-labs/governance-program-library/tree/master/programs/nft-voter

VSR (Voter Stake Registry, aka Mango Locker)
https://github.com/blockworks-foundation/voter-stake-registry